Snippets/SqlPersistence/SqlPersistence_2/SqlPersistenceSettings.cs

```diff
@@ -18,6 +18,7 @@
 
 #region OracleScripts
 [assembly: SqlPersistenceSettings(OracleScripts = true)]
+#endregion
```

Comment:
Those snippets (OracleScripts, PromoteScripts) don't seem to be used anywhere, is that intentional?